docs/execplans/3-9-2-parse-ninja-status-lines-to-drive-task-progress.md

```
@@ -0,0 +1,425 @@
+  — PASSED.
+- Parser unit test
+  `runner::process::streaming::tests::forward_output_with_ninja_status_parses_monotonic_updates`
+  — PASSED.
+- BDD scenario
+  `features_scenarios::progress_output_standard_mode_reports_task_updates_from_ninja_status_lines`
+  — PASSED.
+- BDD scenario
+  `features_scenarios::progress_output_accessible_mode_emits_textual_task_updates`
+  — PASSED.
+- Non-TTY stderr excerpt (stdout redirected):
+  ```text
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)